contrib/docker/testnet.sh

```diff
@@ -8,7 +8,9 @@ pid=$(id -u)
 gid="docker" # Assumes user is in docker group
 
 #Note: sequence limit [2:254]
-sequence="seq -f "%03g" 10 29"
+seq_start=10  # Not 0 because of port assignments
+seq_end=$((${seq_start} + 19))  # TODO(unassigned): arbitrary end amount
+sequence="seq -f "%03g" ${seq_start} ${seq_end}"
 
 #Note: this can avoid to rebuild the docker image
 #custom_build_dir="-v /home/user/kovri:/kovri"
```

Comment:
Huh? But it does work... What do you mean exactly?

Reply:
I was referring to the commented lines

> #Note: this can avoid to rebuild the docker image
> #custom_build_dir="-v /home/user/kovri:/kovri"

The default path for binaries inside the container used to be /kovri before, now it uses the binaries from /usr/bin.

Reply:
>  #custom_build_dir="-v /home/user/kovri:/kovri" 

Oh, you meant `#custom_build_dir="-v /home/user/kovri:/kovri"`.